src/components/orders/DetailsTable/index.tsx

```diff
@@ -82,7 +82,19 @@ const tooltip = {
     'The date and time at which the order was submitted. The timezone is based on the browser locale settings.',
   expiration:
     'The date and time at which an order will expire and effectively be cancelled. Depending on the type of order, it may have partial fills upon expiration.',
-  type: 'An order can be either a Buy or Sell order. In addition, an order may be of type "Fill or Kill" (no partial fills) or a regular order (partial fills allowed).',
+  type: (
+    <div>
+      CoW Protocol supports three type of orders – market, limit and liquidity:
+      <ul>
+        <li>A market order is an order to buy or sell at the market&apos;s current best available price</li>
+        <li>A limit order is an order to buy or sell at an arbitrary price specified by the user</li>
+        <li>A liquidity order is an order that market makers can provide as a source of liquidity</li>
+      </ul>
+      In addition, an order can be &quot;fill or kill&quot; or &quot;partially fillable&quot;. Currently all order types
+      can be &quot;fill or kill&quot; – only market orders can be either &quot;fill or kill&quot; or &quot;partially
+      fillable&quot;.
+    </div>
+  ),
```

Comment:
nit: For every list item, i would add the name of the order in bold, and then explain.

* **Market Order**:  Buy or Sell at ...
* **Limit Order**:  Buy or Sell at ...
* **Liquidity Order**:  Source of liquidity coming from market makers...

Also: 

![image](https://user-images.githubusercontent.com/2352112/203980833-c2ca9a2d-b6de-40ea-8c48-30d64ff78be6.png)


I think this last part is a bit confusing. 

Orders can allow partial execution or not, there's two types:
- **Fill Or Kill**: Either the order is fully filled, or not filled at all. Currently all Market and Limit Orders are fill or kill. Support for partials will come soon!
- **Partial Execution**: The order can be executed partially as long as the limit price is respected.


Anyways, maybe @MindyCoW @avsavsavs have a better input in the specific message. 

As Leandro mentioned, we will need to break this type into 3 different concepts/tags (1. buy/sell, 2. partial/fill-kill, 3. market/limit/liquidity), each could have a tooltip to further clarify.

Reply:
I think this is what made more sense to me: 
`In addition, an order can be "Fill, Kill or Partially Fillable". Currently only market orders are allowed to be partially fillable, while the other orders are just fill or kill.`

But Alex M suggested the copy you have in here[https://cowservices.slack.com/archives/C0361CDG8GP/p1669220826163489?thread_ts=1669206785.099929&cid=C0361CDG8GP](url)

Reply:
there was a problem witht he link above. Adding it again https://cowservices.slack.com/archives/C0361CDG8GP/p1669220826163489?thread_ts=1669206785.099929&cid=C0361CDG8GP


@avsavsavs this is not true `Currently only market orders are allowed to be partially fillable` (it's liquidity orders), but I see is wrong in the original, and AlexM description too

Reply:
Let's settle this topic in the thread of Slack.